test_that_gui_runs_a_minimal_example fails in bleeding

**Describe the bug**
The following test fails the nightly test runs: 
```python
____________ test_that_gui_runs_a_minimal_example[using_job_queue] _____________
[gw2] linux -- Python 3.8.18 /tmp/f_scout_ci/actions-runner-02/_temp/test-kenv/root/bin/python

qtbot = <pytestqt.qtbot.QtBot object at 0x7f4b094c0f70>
storage = <ert.storage.local_storage.LocalStorage object at 0x7f4b094c03a0>
try_queue_and_scheduler = None

    @pytest.mark.scheduler
    @pytest.mark.usefixtures("use_tmpdir", "set_site_config")
    def test_that_gui_runs_a_minimal_example(
        qtbot: QtBot, storage, try_queue_and_scheduler
    ):
        """
        This is a regression test for a crash happening when clicking show details
        when running a minimal example.
        """
        config_file = "minimal_config.ert"
        with open(config_file, "w", encoding="utf-8") as f:
            f.write("NUM_REALIZATIONS 1")
        args_mock = Mock()
        args_mock.config = config_file
    
        ert_config = ErtConfig.from_file(config_file)
        enkf_main = EnKFMain(ert_config)
        with StorageService.init_service(
            project=os.path.abspath(ert_config.ens_path),
        ):
            gui = _setup_main_window(enkf_main, args_mock, GUILogHandler())
            gui.notifier.set_storage(storage)
            qtbot.addWidget(gui)
            start_simulation = gui.findChild(QToolButton, name="start_simulation")
    
            def handle_dialog():
                qtbot.waitUntil(lambda: gui.findChild(QMessageBox) is not None)
                message_box = gui.findChild(QMessageBox)
                qtbot.mouseClick(message_box.button(QMessageBox.Yes), Qt.LeftButton)
    
            QTimer.singleShot(500, handle_dialog)
            qtbot.mouseClick(start_simulation, Qt.LeftButton)
    
            qtbot.waitUntil(lambda: gui.findChild(RunDialog) is not None)
            run_dialog = gui.findChild(RunDialog)
            qtbot.mouseClick(run_dialog.show_details_button, Qt.LeftButton)
>           qtbot.waitUntil(run_dialog.done_button.isVisible, timeout=20000)
E           pytestqt.exceptions.TimeoutError: waitUntil timed out in 20000 milliseconds

/tmp/f_scout_ci/actions-runner-02/_temp/test_root/tests/unit_tests/gui/simulation/test_run_dialog.py:562: TimeoutError
```

**To reproduce**
TBD

**Expected behaviour**
Not fail

**Screenshots**
N/A

**Environment**
 - OS: RHEL7
 - ERT/Komodo release: bleeding
 - Python version: 3.8
 - Remote/HPC execution involved: [yes|no]

**Additional context**
N/A

--- a/tests/unit_tests/gui/simulation/test_run_dialog.py
+++ b/tests/unit_tests/gui/simulation/test_run_dialog.py
@@ -553,4 +553,4 @@ def test_that_gui_runs_a_minimal_example(qtbot: QtBot, storage):
         qtbot.waitUntil(lambda: gui.findChild(RunDialog) is not None)
         run_dialog = gui.findChild(RunDialog)
         qtbot.mouseClick(run_dialog.show_details_button, Qt.LeftButton)
-        qtbot.waitUntil(run_dialog.done_button.isVisible, timeout=20000)
+        qtbot.waitUntil(run_dialog.done_button.isVisible, timeout=200000)

